Monika After Story/game/script-ch30.rpy

```diff
@@ -1170,6 +1170,17 @@ label ch30_end:
 
 # label for things that may reset after a certain amount of time/conditions
 label ch30_reset:
+
+    # setup the docking station to handle the detection
+    $ rpyCheckStation = store.MASDockingStation(renpy.config.gamedir)
+
+    $ listRpy = rpyCheckStation.getPackageList(".rpy")
+
+    if len(listRpy) != 0 and persistent._mas_pm_has_rpy is not True and not config.developer:
```

Comment:
If I quit during the event and then restart, the event will play twice. Need to use `seen_event` here.

Also if the user situation changes (as in, rpys detected, event Queued, user quits during event, removes rpys), on launch we should clear the event from the event_list. **NOTE** this means you will need to clear both the `persistent.event_list` and `persistent.current_monikatopic`.